src/cmake/Modules/FindTileDB_EP.cmake

```diff
@@ -53,9 +53,15 @@ else()
 
     # Try to download prebuilt artifacts unless the user specifies to build from source
     if(DOWNLOAD_TILEDB_PREBUILT)
-        fetch_prebuilt_tiledb(VERSION 2.22.0)
+        fetch_prebuilt_tiledb(
+                VERSION 2.22.0
+                RELLIST_HASH SHA256=6ae0c1558f400cbee68b40810d29ad2cd0398e55378d158a9aafa3cd36b089b1
```

Comment:
I'll add it, generating sha256 is the same for everything: run `sha256sum` however I also added a file `releases.csv.sha256` in the release files so that we don't have to calculate it locally.

Reply:
![Uploading 1713533624_grim.png…]()

Reply:
for reference, it is available under the Assets for each release: https://github.com/TileDB-Inc/TileDB/releases/tag/2.22.0

<img width="529" alt="image" src="https://github.com/TileDB-Inc/TileDB-Vector-Search/assets/327706/6bc390bd-ca24-4de4-9e61-19e936b4d554">